makeflash.sh

```diff
@@ -1,13 +1,14 @@
 #!/bin/sh
 # Usage:
 #
-#      ./makeflash.sh [-C] <output.img>
+#      ./makeflash.sh [-C size] <output.img>
 #
 MKFLASH_TAG="$(linuxkit pkg show-tag pkg/mkflash)-amd64"
 
 if [ "$1" = "-C" ]; then
-    IMAGE=$2
-    dd if=/dev/zero of=$IMAGE count=$(( 1536 * 1024 )) bs=1024
+    SIZE=$2
+    IMAGE=$3
```

Comment:
Do you want to check you have 3 arguments in this case?

Reply:
I guess the question is -- should we really bite the bullet and introduce something like getopts(1) here (or across all of our shell scripts for that matter). For now it feels a bit like an overkill, but let me know if you feel that lacking it makes everything too brittle.